brave/brave-core
# Fixes include/excluded update in the panel

Resolves https://github.com/brave/brave-browser/issues/3480
Resolves https://github.com/brave/brave-browser/issues/3476

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
- enable rewards
- add some sites to ac table
- make sure that exclude/restore works correctly in settings page
- in new window to one page that is in your ac table
- open panel
- try toggling include and make sure that toggle updates and settings page updates
- toggle off current site in the panel
- (don't close panel) click on restore in settings page and make sure that panel reflects this change (toggle is set to include)

## Reviewer Checklist:
